tests/suites/helpers.function

```diff
@@ -8,16 +8,13 @@
 #include "mbedtls/platform.h"
 #else
 #include <stdio.h>
-#define mbedtls_printf     printf
```

Comment:
Is it really necessary to remove the unused macros for `calloc()` and `printf()`. These are useful, basic functions that could be helpful at any point, they do not cause any inefficiencies in the code.